docs/cluster-management/running-locally.md

```diff
@@ -20,3 +21,98 @@ You will be prompted to enter your Client ID and Client Secret. Afterwards, type
 
 [docker]: https://www.docker.com/products/docker
 [docker-compose]: https://www.docker.com/products/docker-compose
+
+## Configuring SD-in-a-Box
+
+SD-in-a-box was intended to be an easy way to run a Screwdriver cluster locally on your development machine so you can demo its features first-hand.
+
+### Custom Docker Images
+
+Since it's powered by Docker, you can determine which images to use for it
+
+Here is a snippet of the `docker-compose.yml` file
+
+```
+version: '2'
+services:
+  api:
+    image: screwdriver-cd/screwdriver:stable
+    . . .
+  ui:
+    image: screwdriver-cd/ui:stable
+    . . .
+  store:
+    image: screwdriver-cd/store:stable
+    . . .
+```
+
+You can make a local Docker image to use instead of one of these.
+
+We could do a blog post on locally building a docker image that extends one of the Screwdriver ones.
```

Comment:
We should add what command to run after they modify the images (for people who are not familiar with docker-compose):
```
docker-compose -p screwdriver up
```